pkgs/development/libraries/libdvdcss/default.nix

```diff
@@ -1,9 +1,13 @@
-{stdenv, fetchurl}:
+{stdenv, fetchurl, IOKit}:
 
-stdenv.mkDerivation rec {
+with stdenv;
+
+mkDerivation rec {
```

Comment:
Hm, not that it makes a huge difference, but this is a bit unusual. What you typically see is `with stdenv.lib`.